### ISSUE TITLE
[BUG] - Accessible name is missing if `Segmented Control` is used with icon only

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3190]
### JS Framework

all

### Porsche Design System version

3.15.0-rc.1

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

all

### Browser version

all

### Reproduction Example

Stackblitz: https://stackblitz.com/edit/ohvkiq?file=index.html

### Steps to reproduce the behavior

1. Go to https://stackblitz.com/edit/ohvkiq?file=index.html
2. Open Dev Console
3. Highlight the `<button>` element
4. Look into accessibility tree
5. See error: the accessible name is empty

### Expected behavior

If icons only are used in `Segmented Control` the user should have the ability to add an corresponding `aria` property to provide an accessible name.

**Hint:** Also re-activate axe-core test in `segmented-control.html` vrt test page!
